webapp/home/subdomains/proteomics/content/database_searching.py

```diff
@@ -0,0 +1,87 @@
+"""Content for data section - tools, workflows and help tabs."""
+
+galaxy_au_support_item = {
+    "title_html": 'Galaxy Australia support',
+    "description_html": """<p>
+            Any user of Galaxy Australia can request support through an
+            online form.
+        </p>""",
+    "button_link": "/request/support",
+    "button_html": "Request support",
+}
+
+tools = [
+    # {   # Accordion item schema:
+    #     "title_html": '',
+    #     "description_html": """""",
+    #     "inputs": [
+    #         {
+    #             'datatypes': [''],
+    #             'label': '',
+    #         },
+    #     ],
+    #     "button_link": "",
+    #     "button_html": "",
+    #     "button_tip": "",
+    #     "view_link": "",
+    #     "view_html": "",
+    #     "view_tip": "",
+    # },
+    {
+        "title_html": '<code>DecoyDatabase</code>',
+        "description_html": '<p>Create decoy sequence database from forward sequence database.</p>',
+        "inputs": [
+            {
+                'datatypes': ['fasta'],
+                'label': 'Input FASTA file(s), each containing a database',
+            }
+        ],
+        "button_link": "https://proteomics.usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/galaxyp/openms_decoydatabase/DecoyDatabase/",
+    },
+    {
+        "title_html": '<code>MaxQuant</code>',
+        "description_html": '<p>MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.</p>',
+        "inputs": [
+            {'datatypes': ['thermo.raw']},
```

Comment:
These first three look like they should be one input with three accepted datatypes, not three separate inputs? This implies that MaxQuant requires a `thermo.raw`, `mzMl` AND `mzXML` input to work! And maybe add a label?